prototype/mixins.py

```diff
@@ -0,0 +1,31 @@
+from django.views.generic import TemplateView
+from django.utils.functional import cached_property
+
+from directory_cms_client.client import cms_api_client
+
+from core.helpers import handle_cms_response
+from prototype.helpers import unprefix_prototype_url
+
+
+class CMSPageViewMixin(TemplateView):
```

Comment:
this is quite similar to core.mixins.GetCMSPageMixin`. it would be nice if we used a similar but different name as that such as `GetCMSPageByUrlPathMixin` or similar. It tells the reader what the mixin does.